### PR TITLE
Fix gid in pvc file

### PR DESCRIPTION
**Pull request by obnoxxx**
### What does this PR achieve? Why do we need it?

When running `heketi-cli volume create --size=1 --persistent-volume-file=pv.json --gid=594`, the gid is not reflected in the output `pv.json` file. This patch fixes this shortcoming.

### Notes for the reviewer

rhbz#1405415

### CODE REVIEW

File: pkg/kubernetes/pv.go [hunk 1]
```diff
@@ -51,5 +53,10 @@ func VolumeToPv(volume *api.VolumeInfoResponse,
 		pv.Spec.Glusterfs.EndpointsName = endpoint
 	}
 
+	gidStr := strconv.FormatInt(int64(volume.Gid), 10)
```
Comment (phlogistonjohn) [inline pkg/kubernetes/pv.go]: Why isn't fmt.Sprintf("%v", volume.Gid) sufficient? Is it because of the type of Gid?

Comment (obnoxxx) [reply pkg/kubernetes/pv.go]: Oh, no deep thought behind it... I was mainly copying it from the kubernetes provisioner code...
Let me try to use Sprintf...

**Comment by Madhu-1** [comment]
Jfyi https://gist.github.com/evalphobia/caee1602969a640a4530 benchmark comparison for sprintf and strconv

**Comment by obnoxxx** [comment]
Interesting! But I don't care too much about performance here, since it's only invoked from the CLI, and in a rather rare case.

**Comment by obnoxxx** [comment]
unrelated, but still strange:
```
13:29:54 === RUN   TestCreateClusterTLSNoVerify
13:29:54 Heketi v8.0.0-348-gb2d14d1-pr_1510
13:29:54 [heketi] INFO 2019/02/06 13:29:54 Loaded mock executor
13:29:54 --- FAIL: TestCreateClusterTLSNoVerify (0.40s)
13:29:54     assert.go:44: 
13:29:54         
ASSERT:	func (github.com/heketi/heketi/tests/functional/TestEnabledTLS/client_tls_test_test.testClientActions) 0x69b4ab
13:29:54         
	File /root/go/src/github.com/heketi/heketi/tests/functional/TestEnabledTLS/client_tls_test/client_tls_test.go:95
13:29:54         
	Info: [expected err == nil, got: Post https://localhost:8080/clusters: dial tcp [::1]:8080: connect: connection refused]
```

**Comment by obnoxxx** [comment]
retest this please

**Comment by obnoxxx** [comment]
Approved and all tests (except openshift-ci) passed. Merging...

**Pull request merged**